PCAxis.Web.Controls/VariableSelector/VariableSelectorValueSelect.ascx

```diff
@@ -79,7 +77,7 @@
     </asp:Panel>
     <asp:Panel ID="HiddenEventButtons" runat="server" Visible="false" CssClass="variableselector_valuesselect_hiddeneventbutton_panel">
     </asp:Panel>
-    <hr class="pxweb-divider type-light with-margin"/>
+    <!--<hr class="pxweb-divider type-light with-margin"/>-->
```

Comment:
Could this be solved by  css. So that we can keep it.

Reply:
The horizontal line is now displayed using CSS instead of hr-element